src/pybamm/util.py

```diff
@@ -23,23 +22,6 @@ def root_dir():
     return str(pathlib.Path(pybamm.__path__[0]).parent.parent)
 
 
-def get_git_commit_info():
-    """
-    Get the git commit info for the current PyBaMM version, e.g. v22.8-39-gb25ce8c41
-    (version 22.8, commit b25ce8c41)
-    """
-    try:
-        # Get the latest git commit hash
-        return str(
-            subprocess.check_output(["git", "describe", "--tags"], cwd=root_dir())
-            .strip()
-            .decode()
-        )
-    except subprocess.CalledProcessError:  # pragma: no cover
-        # Not a git repository so just return the version number
-        return f"v{pybamm.__version__}"
```

Comment:
This function only seemed to be used in tests, and does not seem like it belong in PyBaMM